New run functions

newhoggy opened the pull request.

Kheldar commented:
I am OK with these. Those language extensions are still hilarious, though. 😛

AlexeyRaga commented:
I don't like undecidable instances plus I don't think that we need library support for something as simple as `runResourceT . runAWS e`. 
Even if I needed that, I'd write one little helper that is specific for my stack, don't require Antiope's help here.

newhoggy commented:
I've removed the language extensions I didn't need.

I just dropped all the the languages extensions I found in the `generic-lens` module I was trying to imitate because I wanted to write the code fast.

Also, I've add "THE ONE" support.
